src/thunderbots/software/backend/input/network/networking/BUILD

```diff
@@ -0,0 +1,63 @@
+package(default_visibility = ["//visibility:public"])
+
+cc_library(
+    name = "ssl_gamecontroller_client",
+    srcs = ["ssl_gamecontroller_client.cpp"],
+    hdrs = ["ssl_gamecontroller_client.h"],
+    #    visibility = ["//visibility:private"],
+    deps = [
+        "//software/proto:ssl_cc_proto",
+        "@g3log//:all",
+    ],
+)
+
+cc_library(
+    name = "ssl_vision_client",
+    srcs = ["ssl_vision_client.cpp"],
+    hdrs = ["ssl_vision_client.h"],
+    #    visibility = ["//visibility:private"],
+    deps = [
+        "//software/proto:ssl_cc_proto",
+        "@g3log//:all",
+    ],
+)
+
+cc_library(
+    name = "network_filter",
+    srcs = ["network_filter.cpp"],
+    hdrs = ["network_filter.h"],
+    deps = [
+        "//software/ai/world:ball",
+        "//software/ai/world:field",
+        "//software/ai/world:refbox_constants",
+        "//software/ai/world:team",
+        "//software/backend/input/network/filter:ball_filter",
+        "//software/backend/input/network/filter:robot_filter",
+        "//software/backend/input/network/filter:robot_team_filter",
+        "//software/proto:ssl_cc_proto",
+        "//software/util/parameter:dynamic_parameters",
+    ],
+)
+
+cc_library(
+    name = "network_client",
+    srcs = [
+        "network_client.cpp",
+        # TODO: We shouldn't have to have the *_client.cpp files here, but full_system
```

Comment:
have you got a chance to look into this?

Reply:
Yup, this is fixed now (just removed the files and it was good).